src/solver/constraints-builder/cbuilder.cpp

```diff
@@ -392,11 +392,6 @@ bool CBuilder::completeCBuilderFromFile(const String& filename)
                     key = p->key;
                     key.toLower();
 
-                    if (key == "study")
-                    {
-                        pStudyFolder = p->value;
-                        continue;
-                    }
```

Comment:
Why remove this ?

Reply:
variable `pStudyFolder` is unused
on this note we should decide what to do with the constraint builder as it's not used by antares web right now